backend/de.metas.manufacturing.rest-api/src/main/java/de/metas/manufacturing/job/service/commands/ReceiveGoodsCommand.java

```diff
@@ -54,13 +60,16 @@
 	// State
 	private I_PP_Order _ppOrder; // lazy
 	private I_PP_Order_BOMLine _coProductLine;
+	private IMutableHUContext huContext;
```

Comment:
i think huContext is no longer used (along with huContextFactory)

Reply:
cleaned out